guides/common/modules/proc_configuring-tracer-on-a-host.adoc

```diff
@@ -0,0 +1,28 @@
+[id="configuring-tracer-on-a-host_{context}"]
+= Configuring Tracer on a host
+
+You can install Tracer on hosts and access Traces on {Project}.
+Tracer displays a list of services and applications that are outdated and need to be restarted.
+Traces is the output generated by Tracer in the {ProjectWebUI}.
+
+Tracer is supported on all operating systems that are supported on {Project}.
+
+.Prerequisites
+* The host is registered to {ProjectName}.
+include::snip_prerequisite-project-client-repository-enabled.adoc[]
+* Remote execution is enabled.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *All Hosts*.
+. Click the name of the host you want to modify.
+. On the *Traces* tab, click *Enable Traces*.
+. Select the provider to install `katello-host-tools-tracer` from the list.
+. Click *Enable Tracer*.
+You get a REX job notification after the remote execution job is complete.
+. Start a remote execution job to upload the Tracer data from your host to {Project}:
++
+For more information, see xref:Configuring_and_Setting_Up_Remote_Jobs_{context}[].
++
+----
+# katello-tracer-upload
```

Comment:
Maybe its better to document, the "Resolve Traces" REX job which is normally used to a) reboot and b) to restart the services (which need a restart)

Reply:
@sbernhard Are we talking about these jobs?
`Resolve Traces - Katello Ansible Default`
`Resolve Traces - Katello Script Default`

@maximiliankolb Do you suggest that we just mention here that the user can also run a REX job, something like:
`Alternatively, you can run a *Resolve Traces* job from the Job Templates page in the WebUI.` or also fully document how to launch such a job by filling out the form? This WebUI job launching seems to be more out of scope of this procedure rather than not.

Reply:
yes

Reply:
This is how it looks like, if you have traces and have the possibility to restart apps:

![image](https://github.com/user-attachments/assets/53bcb432-d819-4373-b1d2-2dda1929b142)

Reply:
If you press "customized rex", you will end up in a REX execution summary like:

![image](https://github.com/user-attachments/assets/c6473ac2-76a7-4c66-98ef-de6a96bbc5e8)

Reply:
@sbernhard Thank you! Does the procedure look good now?
@maximiliankolb Do you have any further feedback? Thanks.